libusb/os/windows_winusb.c

```diff
@@ -1409,6 +1409,15 @@ static int set_composite_interface(struct libusb_context *ctx, struct libusb_dev
 	priv->usb_interface[interface_number].path = dev_interface_path;
 	priv->usb_interface[interface_number].apib = &usb_api_backend[api];
 	priv->usb_interface[interface_number].sub_api = sub_api;
+
+	{
+		int ii;
```

Comment:
Just "i" is the norm for such loop counters. "i" can also be thought of as a very short form of "interfaces".